products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -707,15 +709,15 @@ public final class BuildSystem {
         info.pointee.inode = UInt64(s.st_ino)
         info.pointee.mode = UInt64(s.st_mode)
         info.pointee.size = UInt64(s.st_size)
-        #if os(macOS)
```

Comment:
Please change this to `#if canImport(Darwin)` as well

Reply:
Done.